swarm_test.go

```diff
@@ -194,6 +182,85 @@ func TestNewSwarm(t *testing.T) {
 	}
 }
 
+func TestNewSwarmFailure(t *testing.T) {
```

Comment:
Add a description comment of what this test is doing.
A reader should not first have to read the code to understand what a test is doing.
(Exceptions could be tests which are absolutely clear from the name what they test - I claim this does not apply to this test)

Reply:
Fixed.